src/main/java/core/basesyntax/MyLinkedList.java

```diff
@@ -1,47 +1,206 @@
 package core.basesyntax;
 
 import java.util.List;
+import java.util.Objects;
 
 public class MyLinkedList<T> implements MyLinkedListInterface<T> {
+
+    class Node<T> {
```

Comment:
```suggestion
    private class Node<T> {
```

давай добавим модификатор доступа